Sources/TSCBasic/Process.swift

```diff
@@ -436,16 +456,27 @@ public final class Process: ObjectIdentifierProtocol {
         var outputPipe: [Int32] = [0, 0]
         var stderrPipe: [Int32] = [0, 0]
```

Comment:
nit: I'd recommend putting [-1, -1] as the default values.

Reply:
changed to -1